MiniBlog/Services/ArticleService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using MiniBlog.Stores;
+using MiniBlog.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MiniBlog.Services
+{
+    public class ArticleService
+    {
+        private readonly IUserStore userStore;
+        private readonly IArticleStore articleStore;
+        public ArticleService(IArticleStore articleStore, IUserStore userStore)
```

Comment:
do we need inject userstore to article service?

Reply:
> do we need inject userstore to article service?

case by case, but better to change naming

Reply:
agree!